bouvet-deler/databricks_lambda/databricks_lambda_case2.py

```diff
@@ -0,0 +1,76 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Using Lambda in Databricks - Case 2
+# MAGIC 
+# MAGIC This notebooks gives a slightly complex example of using lambda in Databricks. For more information see this article (add link).
+# MAGIC 
+# MAGIC Author: Morten Mjelde <morten.mjelde@bouvet.no>
+
+# COMMAND ----------
+
+# Define DataSet3
+from pyspark.sql import Row
+
+DataSet3 = spark.createDataFrame(
+    [
+        Row(location="Oslo", people=[Row(firstName="Harald", lastName="Haraldsen"), Row(firstName="Mark", lastName="Hunter")]),
```

Comment:
```suggestion
        Row(location="Oslo", people=[
                Row(firstName="Harald", lastName="Haraldsen"), 
                Row(firstName="Mark", lastName="Hunter")
            ]),
```
Maybe for readabilty?

Reply:
Fixed.